packages/vue-flicking/src/Flicking.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+
+import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS } from "@egjs/flicking";
+import ChildrenDiffer from "@egjs/vue-children-differ";
+import ListDiffer, { DiffResult } from "@egjs/list-differ";
+import { Component, Vue, Prop } from "vue-property-decorator";
+import { CreateElement, VNodeData, VNode } from "vue";
+import { merge } from "./utils";
+
+@Component({
+  name: "Flicking",
+  directives: {
+    "children-differ": ChildrenDiffer,
+  },
+})
+export default class Flicking extends Vue {
+  // Tag of wrapper element
+  @Prop({ type: String, default: "div", required: false }) tag!: string;
+  @Prop({ type: Object, default: () => ({}), required: false }) options!: Partial<FlickingOptions>;
+  @Prop({ type: Array, default: () => ([]), required: false }) plugins!: Plugin[];
+
+  // Following decorator will inject native Flicking's method into Vue-Flicking
+  @withFlickingMethods
+  private $_nativeFlicking!: NativeFlicking;
+  private $_pluginsDiffer!: ListDiffer<Plugin>;
+  private $_cloneCount!: number;
+
+  public mounted() {
+    this.$_pluginsDiffer = new ListDiffer<Plugin>();
+    this.$_cloneCount = 0;
+
+    const options = merge({}, this.options, { renderExternal: true });
```

Comment:
How about `{...this.options, ...{renderExternal: true}}`?

Reply:
That would be perfect. I'll fix it.